src/app/components/administration/forecast/forecast-edit/forecast-edit.component.ts

```diff
@@ -45,7 +45,20 @@ export class ForecastEditComponent implements OnInit {
   }
 
   updateForecast() {
-
+    if (this.editForecastForm.valid) {
+      this.forecastService.updateForecast({
+        id: 1,
+        title: this.forecast.title,
+        dateTime: this.forecast.dateTime,
+        description: this.editForecastForm.value.description,
```

Comment:
set all parameters to same values except description. description parameter gets form value